pkg/controllers/dataexport/reconcile.go

```
@@ -1502,7 +1503,7 @@ func waitForPVCBound(in kdmpapi.DataExportObjectReference, checkMounts bool) (*c
 
 	if wErr != nil {
 		logrus.Errorf("%v", wErr)
-		return nil, wErr
+		return nil, err
```

can we return the error message also in wErr , such that from the final message it will be "timeout happened with the %v error), err"

---

Fixed.